docs/docsite/rst/user_guide/vault.rst

```diff
@@ -37,8 +54,17 @@ Vault IDs and Multiple Vault Passwords
 *Available since Ansible 2.4*
 
 A vault ID is an identifier for one or more vault secrets. Since Ansible 2.4,
-Ansible supports multiple vault passwords. Vault IDs provide
-labels for individual vault passwords.
+Ansible supports multiple vault passwords.
+
+Vault IDs provide labels to distinguish between individual vault passwords.
+
+To use vault IDs, you must provide an ID *label* of your choosing and a *source* to obtain its password (either ``prompt`` or a file path):
+
+.. code-block:: bash
+
+   --vault-id label@source
+
+This switch is available for all Ansible commands that can interact with vaults: :ref:`ansible-vault`, :ref:`ansible-playbook`, etc.
 
 Vault-encrypted content can specify which vault ID it was encrypted with.
 
```

Comment:
I think the two paragraphs below should also be amended.

It is not important the reader learns about the history of ansible-vault and Ansible. This documentation should be written for someone coming to it fresh, using the current version of Ansible.

I think these paragraphs should be replace by a short note at the end of the section, something like:

```
.. note:
    Older versions of Ansible, before 2.4, only supported using one single vault password at a time. 
```

Reply:
Hello,

Unless I've mis-githubbed it should be done.